servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/LoadBalancingPolicies.java

```diff
@@ -18,9 +18,9 @@
 /**
  * A collections of factories for constructing a {@link LoadBalancingPolicy}.
  */
-public final class LoadBalancerPolicies {
+public final class LoadBalancingPolicies {
```

Comment:
Wanna let 1 version to go with deprecations?

Reply:
Yes, let's do one release.